scripts/preLoading.st

```diff
@@ -18,7 +18,8 @@ MetacelloPharoPlatform select.
     'Iceberg-TipUI'
     'Iceberg-Plugin-Pharo' 
     'Iceberg-Plugin-Metacello' 
-    'Iceberg-Plugin-GitHub' 
+    'Iceberg-Plugin-GitHub'
+    'Iceberg-Plugin-Migration'
```

Comment:
This should also be in the update script of the README IMO. It will be needed after the integration of this code in Pharo.

Reply:
Yup. Not for Pharo6, but yes for Pharo7.